catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -29,7 +29,7 @@
 import org.slf4j.LoggerFactory;
 
 import ddf.catalog.Constants;
-import ddf.catalog.core.versioning.MetacardVersion;
+import ddf.catalog.core.versioningapi.MetacardVersion;
```

Comment:
change this package to `versioning` and the MetacardVersionImpl package to `versioning.impl`